app/models/proposal.rb

```diff
@@ -60,10 +60,7 @@ class Proposal < ActiveRecord::Base
   scope :for_state, ->(state) do
     where(state: state).order(:title).includes(:event, {speakers: :user}, :review_taggings)
   end
-  scope :in_track, ->(track) do
-    track = nil if track.try(:strip).blank?
-    where(track: track)
-  end
```

Comment:
I'll take a shot at putting the scope back, and being more explicit with a `track_id` param in EventStats. It made me nervous to change this, as I felt there could be ripple effects I wouldn't see. One thing I came across is that when 'All' is selected from the dropdown, the 'Accepted Waitlisted' stats to the right disappear. I don't think that is expected behavior, and my change didn't remedy it. So I added an Unstaged card for it.